app/routes/payments/$paymentId.tsx

```diff
@@ -1,12 +1,35 @@
+import type { ReactElement } from 'react';
 import * as React from 'react';
 import type { MetaFunction } from '@remix-run/node';
 import { LoaderFunction } from '@remix-run/server-runtime';
 import invariant from 'tiny-invariant';
 import { useLoaderData } from '@remix-run/react';
 import { useTranslation } from 'react-i18next';
-import { Typography } from '@mui/material';
-import { Page } from '@numaryhq/storybook';
+import { Box, Chip, Divider, Typography } from '@mui/material';
+import {
+  Date,
+  DividerWithSpace,
+  Page,
+  Row,
+  Table,
+  theme,
+} from '@numaryhq/storybook';
 import ComponentErrorBoundary from '~/src/components/Wrappers/ComponentErrorBoundary';
+import ContentCopy from '@mui/icons-material/ContentCopy';
+import StopIcon from '@mui/icons-material/Stop';
+import { JSONTree } from 'react-json-tree';
+import { AdjustmentsItem, PaymentDetail } from '~/src/types/payment';
+import { API_PAYMENT, ApiClient } from '~/src/utils/api';
+import { copyTokenToClipboard } from '~/src/utils/clipboard';
+
+// this will be deleted when Reconciliation is done
```

Comment:
can you add `// TODO` to be able to use our IDE to find and fix todo ? (it's a good reminder !)